fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -442,6 +442,8 @@
 
     private long imageJournalId;
 
+    private long feStartTime = System.currentTimeMillis();
```

Comment:
Better to set feStartTime after all service started. https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java#L130
In the case where there is logs of bdb log to replay, the delay between the start of the fe process and the start of the service can be several minutes.

Reply:
Done, So the feStartTime is not the fe process start time, but the time to start providing services